SDK_dotnet/Models/Exchange/AcceptQuoteRequest.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace Gluwa.SDK_dotnet.Models.Exchange
+{
+    public sealed class AcceptQuoteRequest
+    {
+        /// <summary>
+        /// Checksum that was received when quote was created.
+        /// </summary>
+        [Required]
+        public string Checksum { get; set; }
+
+        /// <summary>
+        /// All orders that will fulfill this quote.
+        /// </summary>
+        [Required]
+        public List<MatchedOrderForQuote> MatchedOrders { get; set; }
+
+        public IEnumerable<ValidationResult> Validate(MatchedOrderForQuote matchedOrders, ECurrency currency)
```

Comment:
why do you need matchedOrders as a parameter? You already have access to it via MatchedOrders property